BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsClient.kt

```diff
@@ -210,6 +210,7 @@ internal class AnalyticsClient constructor(
             .put(FPTI_KEY_EVENT_NAME, event.name)
             .put(FPTI_KEY_TIMESTAMP, event.timestamp)
             .put(FPTI_KEY_VENMO_INSTALLED, event.venmoInstalled)
+            .put(FPTI_KEY_PAYPAL_INSTALLED, event.isPayPalInstalled)
```

Comment:
Hey Rich. Some of this logic has changed recently. Here's [the PR](https://github.com/braintree/braintree_android/pull/1104) with the changes. We decided to start putting `venmo_installed` and `paypal_installed` into batch params now since they aren't tied to any individual event.

Reply:
😲 Thanks for the heads up!!!

Reply:
Yeah no prob it just got merged in. It's a lot cleaner now! We were able to remove the `DeviceInspector` reference from `BraintreeClient` in the updated implementation.

Reply:
Updated!!! I had to add the `DeviceInspector` to `BraintreeClient` 😔 because I need it to check if the PayPal app is installed. This way, we can determine the AppSwitch flow. Let me know if there's any other approach we can take.